float8_experimental/float8_dynamic_linear.py

```diff
@@ -72,40 +74,117 @@ def forward(self, x):
         y = cast_to_float8_e5m2_bw(y, self.backward_config)
```

Comment:
this line doesn't make sense for inference

Reply:
to clarify, I know it does nothing, I'm just saying it's a bit confusing and requires the reader to think more to understand what's going on